src/test/functional/features/form-failures.feature

```diff
@@ -294,7 +294,7 @@ Feature: Form failures
     And I click "I believe that the facts stated in this application are true"
     When I click "Continue to payment"
     And I pay and submit the application
-    Then the page should include "Application submitted"
+    Then the page should include "Application saved"
```

Comment:
I think it could be worth having a tech debt to update functional tests so we use element IDs for assertions. I've done a quick update to the text in this PR to get CI passing